databricks/platform/platform.go

```diff
@@ -0,0 +1,46 @@
+package platform
+
+import (
+	"errors"
+	"fmt"
+)
+
+//go:generate go run github.com/raito-io/enumer -type=DatabricksPlatform -trimprefix=DatabricksPlatform -transform=lower
+type DatabricksPlatform int
+
+const (
+	DatabricksPlatformAzure DatabricksPlatform = iota + 1
+	DatabricksPlatformGCP
+	DatabricksPlatformAWS
+)
+
+func (p DatabricksPlatform) Host() (string, error) {
+	url, err := p.fmtUrl()
```

Comment:
Better call the WorkspaceAddress method here instead of duplicating all the same lines of code